imi/src/main/resources/imi.properties

```diff
@@ -17,6 +17,14 @@ imi.max_query_block=1000
 #4.3.1 TARGET_FILE Notification API
 imi.target_file_notification_url=http://localhost/obdmanager/notifytargetfile
 
+#url to use to notify the IVR system that the CDR file they sent was processed
+#4.3.2 CDRFileProcessedStatus Notification API
+imi.cdr_file_notification_url=http://localhost/obdmanager/NotifyCDRFileProcessedStatus
+
 #4.4.1 Target File Format
 #field #2 ServiceId - Unique Id provided by IMImobile for a particular service
-imi.target_file_imi_service_id=imiserviceid
\ No newline at end of file
+imi.target_file_imi_service_id=imiserviceid
+
+#maximum number of errors allowed in a CDR file after which all errors are ignored so as to not overwhelm the
+#audit table & the tomcat log file
+imi.max_cdr_error_count=100
```

Comment:
We had discussed putting operations based config in files and other config in the db.  I'm not sure where this one falls.